geospace-http/src/test/scala/com/socrata/geospace/http/GeospaceServletSpec.scala

```diff
@@ -60,7 +82,8 @@ class GeospaceServletSpec extends FakeSodaFountain {
   }
 
   // NOTE: This also tests reprojection, since a non-WGS84 shapefile is ingested
-  test("points geocode properly with cache loaded from local shapefile") {
+  // NOTE: loading from local shapefile is not going to work anymore due to partitioning
+  ignore("points geocode properly with cache loaded from local shapefile") {
```

Comment:
Yeah, see the comment.  I could just remove the test and the route, maybe I'll do that.